### ISSUE TITLE
Buttons not rendering on Edge/Windows 10

### ISSUE THREAD

**Issue by DavidWhaleMEF** [Microsoft/pxt#2168]
microbit.org-ticket: 1882

Hello all,
since some days the micor:bit PXT Eeditor shows no icons anymore (see attaches Screen shot) and I cannot confirm the cooky message. I'm using Windows 10 with Microsoft Edge. How can I solve that?
Thanks in advance

![mb](https://cloud.githubusercontent.com/assets/25079586/26490884/1a54ebfc-4205-11e7-85e6-3267aaabf119.jpeg)

**Comment by DavidWhaleMEF** [comment]
Seen before in: https://github.com/Microsoft/pxt/issues/1872

**Comment by abchatra** [comment]
Please share the edge version

**Comment by DavidWhaleMEF** [comment]
Will do, already asked customer for version number, will update you when we hear back.

**Comment by pelikhan** [comment]
Abchatra we need to pick a minimum version and refuse to load older ones.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by DavidWhaleMEF** [comment]
This is a good idea @pelikhan - also if you can update your 'supported browsers' documentation for PXT to include what the minimum supported version is of each browser, that way we can always use that to advise customers to upgrade. 

I do like the idea that either you refuse to load PXT, or you pop up a warning box saying something like the user is using an older browser and you suggest they update.

The best UX here might be to redirect old browser users to a page saying they are using an older unsupported browser, listing the minimum versions we support, and giving them a link to 'load the editor anyway' with a disclaimer that not all features might function as expected unless they update to (minimum version)?

Some schools might have less than the minimum version installed, and it might be really hard for them to update all browsers on all machines within term time - so blocking old browsers would be a bad experience for them as they would no longer be able to use the editors, although pushing ahead past a warning and expecting there to be degraded performance will be fine for them I think.

So, really like the idea, but let's get the UX of this right so we don't lock out a large proportion of our active user base in the process? Thanks!

**Comment by DavidWhaleMEF** [comment]
this is the current Version of Windows and Edge:
 
Windows 10 with "Creators Update" and last patch level

**Comment by abchatra** [comment]
@DavidWhaleMEF  I would like to get the following information:

Go to ...->settings->About this app

Microsoft Edge 38.14393.1066.0
Microsoft EdgeHTML 14.14393

I am not sure how to make this actionable without try to repro this bug.

**Comment by DavidWhaleMEF** [comment]
@abchatra Sure, contacting customer now. Would you consider a screen share session with the customer that has the issue, perhaps?

**Comment by abchatra** [comment]
Yes sure.

**Comment by DavidWhaleMEF** [comment]
User has found a workaround, but here are the details in case they help you track this down...


Microsoft Edge 40.15063.0.0
Microsoft EdgeHTML 15.15063
 
Meanwhile I found a workaround.
From this site http://www.microbit.org/code/ I start the editor with [Let's Code].
The following URL will be opened but with the problem described ...
https://pxt.microbit.org/?lang=en
Now I edited the URL ... delete "en", type it again (en), press "enter" and everything is fine.
I guess there is something wrong with the "automatic" language setting. I use a german OS/Edge (culture: de-DE)
